docs/DafnyRef/Grammar.md

```diff
@@ -464,9 +466,9 @@ TypeVariableName = NoUSIdent
 MethodName = NoUSIdent
 FunctionName = NoUSIdent
 PredicateName = NoUSIdent
-LabelName = NoUSIdent
+LabelName = NoUSIdentOrDigits
 AttributeName = NoUSIdent
-FieldIdent = NoUSIdent
+FieldIdent = NoUSIdent // TODO ????
```

Comment:
Should this TODO still be here?

Reply:
Yes - I'll get to the types in the next round of edits. FieldIdent is not actually used in the grammar -- just at the moment in the RM